scripts/in_container/run_prepare_provider_packages.sh

```diff
@@ -196,9 +197,9 @@ function rename_packages_if_needed() {
         fi
     fi
 
-    popd
+    popd >/dev/null
     echo
-    echo "Airflow packages are in dist folder "
+    echo "${COLOR_GREEN_OK} Airflow packages are prepared in dist folder${COLOR_RESET}"
```

Comment:
```suggestion
    echo "${COLOR_GREEN}OK Airflow packages are prepared in dist folder${COLOR_RESET}"
```

Reply:
Apologies, this is needed coz I just merged https://github.com/apache/airflow/commit/9d6c7487dd03a42d1b6a1f19fae0c9a746139dbe

Reply:
Nohing to apologise for :).